src/runtime_src/core/common/utils.cpp

```diff
@@ -180,6 +180,26 @@ unit_convert(size_t size)
   return ret+" "+unit[i-1];
 }
 
+std::string
+num_to_hexstring(uint64_t size)
+{
+  std::stringstream sstream;
+  sstream << std::hex << size;
+  std::string result = sstream.str();
+  //return result;
+  return "0x"+result;
+}
+
+uint64_t
```

Comment:
The method std::stoll might this operation easier:
[http://www.cplusplus.com/reference/string/stoll/](http://www.cplusplus.com/reference/string/stoll/)

Reply:
done